apps/backend/src/modules/categories/category.repository.ts

```diff
@@ -92,7 +92,10 @@ class CategoryRepository implements Repository {
 	}
 
 	public async findAll(): Promise<CategoryEntity[]> {
-		const categories = await this.categoryModel.query().select("*");
+		const categories = await this.categoryModel
+			.query()
+			.select("*")
+			.withGraphJoined(RelationName.SCORES);
```

Comment:
Does it mean that all user scores will be joined? Or only for the current user? Could you share a response?

Reply:
![Screenshot (345)](https://github.com/user-attachments/assets/58024611-cff6-45c4-b23e-fbe1b0a1e9cd)

Reply:
Here's responses for `CategoryEntity.toObject()` without mapping.

![Screenshot (346)](https://github.com/user-attachments/assets/f87eac6e-a68d-4048-9738-7a9233e732a8)

Reply:
I notice that `id` from scores response suppose to be named "userId" because I can't find 5 or 6 id on the `quiz_answers_to_users` table.

Reply:
We don't want to send scores for all users to the client. Let's go back to a previous approach, where you separated `findAll` and `findAllWithScores`, where `findAllWithScores` has the scores for current user only